### PR TITLE
Tracing without Performance

### LINKED ISSUE #713: Tracing without Performance
This will add Tracing without Performance to the Laravel SDK, a follow-up of https://github.com/getsentry/sentry-php/issues/1525.

### PR DESCRIPTION
**Pull request by cleptric**
Closes #713

### CODE REVIEW

File: src/Sentry/Laravel/Integration.php [hunk 1]
```diff
@@ -192,13 +189,7 @@ public static function sentryTracingMeta(): string
...
     public static function sentryBaggageMeta(): string
     {
-        $span = SentrySdk::getCurrentHub()->getSpan();
-
-        if ($span === null) {
-            return '';
-        }
-
-        return sprintf('<meta name="baggage" content="%s"/>', $span->toBaggage());
+        return sprintf('<meta name="baggage" content="%s"/>', getBaggage());
```
Comment (cleptric) [inline src/Sentry/Laravel/Integration.php:192]: As we'll very likely always get some value here, I think it's OK to omit checks here.

**Comment by cleptric** [comment]
@stayallive the last TODO here would be to check where we would need to create a new propagation context, especially in a queue context. Today, we control this via `queue_job_transactions`, so maybe we can move this flag into `queue_job_traces` or something.

**Review by stayallive — APPROVED** [review]
This looks 🍑-y to me!

**Comment by cleptric** [comment]
The propagation context does not concern us that much in the case of transactions. It's more about which `traceId` is used for error events. 

I opted to attach a new `PropagationContext` to the span once `pushScope()` is called, hence linking all errors that happen in this specific queue job together.
In case the payload contains a traceparent and baggage payload, and the SDK is configured for `queue_job_transactions`, the `PropagationContext` on the scope will be overwritten by `continueTrace()`, hence continuing the trace with the correct `traceId`.

**Comment by stayallive** [comment]
Good call, that should be fine!

**Review by stayallive — APPROVED** [review]

**Pull request merged**